squidpy/pl/_interactive/_model.py

```diff
@@ -63,7 +63,7 @@ def __post_init__(self) -> None:
         except KeyError:
             raise KeyError(
                 f"Unable to subset the image container with library ids `{self.library_id}`. "
-                f"Valid library ids are `{self.container.library_ids}`."
+                f"Valid container library ids are `{self.container.library_ids}`."
```

Comment:
just because user might do this
```python
img.interactive(adata, library_key="library_id")
```

which would throw the same error although simpli library_id wasn't specified.